src/core/client/admin/ui/components/Label/Label.css

```diff
@@ -0,0 +1,8 @@
+.root {
+  color: var(--v2-palette-input-label);
+  font-size: var(--v2-font-size-2);
+  line-height: var(--v2-lh-reset);
```

Comment:
Did we add a variant to allow use of `lh` as well as `line-height`?
If you want to use this more shorthand term, go ahead, but make sure to add it to the `variables2`.